qf_lib/common/utils/miscellaneous/parabolic_sar.py

```diff
@@ -0,0 +1,70 @@
+#     Copyright 2016-present CERN – European Organization for Nuclear Research
+#
+#     Licensed under the Apache License, Version 2.0 (the "License");
+#     you may not use this file except in compliance with the License.
+#     You may obtain a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#     Unless required by applicable law or agreed to in writing, software
+#     distributed under the License is distributed on an "AS IS" BASIS,
+#     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#     See the License for the specific language governing permissions and
+#     limitations under the License.
+
+from qf_lib.common.enums.price_field import PriceField
+from qf_lib.containers.dataframe.prices_dataframe import PricesDataFrame
+from qf_lib.containers.series.qf_series import QFSeries
+
+
+def parabolic_sar(df: PricesDataFrame, acceleration=0.02, max_acceleration=0.2):
+    """Computes the parabolic SAR (Stop And Return) indicator for the given dataframe.
+
+    Parameters
+    ----------
+    df: PricesDataFrame
+        PricesDataFrame containing High, Low, Close PriceFields.
+    acceleration: float
+        Acceleration factor - higher values keep psar values closer to the price line.
+    max_acceleration: float
+        Limit for the acceleration factor.
+
+    Returns
+    -------
+    QFSeries
+        Series with PSAR values for each index in the dataframe.
+    """
+    df.dropna(inplace=True)
```

Comment:
Let's not change the original df and not do the "inplace" dropna.
Specify what do we want to dropna on (which columns)